.chloggen/sdr-num.yaml

```
@@ -0,0 +1,30 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type:
+  bug_fix
```

this is an enhancement since it adds a new option
```suggestion
change_type: enhancement
```